src/scan.ts

```diff
@@ -18,27 +18,56 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+import { handleJREProvisioning, serverSupportsJREProvisioning } from './java';
 import { log, LogLevel, setLogLevel } from './logging';
 import { getPlatformInfo } from './platform';
 import { getProperties } from './properties';
-import { ScannerProperty, ScanOptions } from './types';
+import { ScannerProperty, JreMetaData, ScanOptions } from './types';
+import { version } from '../package.json';
 
 export async function scan(scanOptions: ScanOptions, cliArgs?: string[]) {
   const startTimestampMs = Date.now();
   const properties = getProperties(scanOptions, startTimestampMs, cliArgs);
+
+  const serverUrl = properties[ScannerProperty.SonarHostUrl];
+  const explicitJREPathOverride = properties[ScannerProperty.SonarScannerJavaExePath];
+
   if (properties[ScannerProperty.SonarVerbose] === 'true') {
     setLogLevel(LogLevel.DEBUG);
     log(LogLevel.DEBUG, 'Setting the log level to DEBUG due to verbose mode');
   }
 
+  if (properties[ScannerProperty.SonarLogLevel]) {
+    setLogLevel(properties[ScannerProperty.SonarLogLevel]);
+    log(LogLevel.DEBUG, `Overriding the log level to ${properties[ScannerProperty.SonarLogLevel]}`);
+  }
```

Comment:
my thought was log level is more specific, with values other than DEBUG. I would defer this to the experts :D. this would only ever happen too if both were set, and they would see two outputs

```
[DEBUG] Bootstrapper::  Setting the log level to DEBUG due to verbose mode
[DEBUG] Bootstrapper::  Overriding the log level to DEBUG
```